amundsen_application/static/js/ducks/tableMetadata/api/v0.ts

```diff
@@ -18,41 +18,20 @@ export type TableDataAPI= { tableData: TableData; } & MessageAPI;
 
 /** HELPERS **/
 import {
-  getTableQueryParams, getTableDataFromResponseData, getTableOwnersFromResponseData, getTableTagsFromResponseData,
+  getTableQueryParams, getTableDataFromResponseData, getTableOwnersFromResponseData,
   createOwnerUpdatePayload, createOwnerNotificationData, shouldSendNotification
 } from './helpers';
 
-export function getTableTags(tableKey: string) {
-  const tableParams = getTableQueryParams(tableKey);
-  return axios.get(`${API_PATH}/table?${tableParams}`)
-  .then((response: AxiosResponse<TableDataAPI>) => {
-    return getTableTagsFromResponseData(response.data);
-  });
-}
-
-/* TODO: Typing this method generates redux-saga related type errors that needs more dedicated debugging */
-export function updateTableTags(tagArray, tableKey: string) {
-  const updatePayloads = tagArray.map((tagObject) => {
-    return {
-      method: tagObject.methodName,
-      url: `${API_PATH}/update_table_tags`,
-      data: {
-        key: tableKey,
-        tag: tagObject.tagName,
-      },
-    }
-  });
-  return updatePayloads.map(payload => { axios(payload) });
-}
 
+// TODO - Revisit the necessity of filtering out tags from main metadata
```

Comment:
The idea was to have as little bloat as possible in the application state. I didn't find it reasonable to set things up such that `globalState.tableMetadata.tableData.tags` and `globalState.<tag-state>.tags` would store the same information given that information could be a list of objects of unknown length.